mirascope/base/extractors.py

```diff
@@ -66,6 +73,18 @@ async def extract_async(self, retries: int = 0) -> ExtractedTypeT:
         """Asynchronously extracts the `extraction_schema` from an LLM call."""
         ...  # pragma: no cover
 
+    # @abstractmethod
```

Comment:
why commented?

Reply:
This would require adding it to every extractor, but few support streaming tools, so leaving commented out until we expect all providers to have this functionality (i.e. don't want to have `stream` implemented for `MistralExtractor` but just throw `NotImplementedError`.

Reply:
put comment explaining abv

Reply:
Done.